autocrop/cli.py

```diff
@@ -188,6 +199,14 @@ def confirmation(question):
         notification_str = "Please respond with 'y' or 'n'"
         print(notification_str)
 
+def chk_extension(p):
+    """Check if the extension passed is valid or not."""
+    inp_extension = str(p)
```

Comment:
A couple things here.

1. Could you use a more descriptive variable name here instead of `p`? e.g. `extension`.
2. You can condense this line to `str(extension).lower()`
3. Adding
```py
if not extension.startswith('.'):
    extension = f".{extension}"
```
would let you make a single `COMBINED_FILETYPES` check below instead of the `elif`.

Reply:
Make sure you run `make check` before submitting your commits, the linter will tell you what to change. That's what's making CI fail at the moment.

Reply:
One last thing! Could you write a couple tests to make sure this new feature is robust? Nothing fancy; you can copy from what's already in `tests/test_cli.py`

Cheers! 💪

Reply:
This also done, will pull the request soon

Reply:
i'm not getting how to write `tests/test_cli.py` can you do it youreself